user-guide/Advanced_Modules/Correlation/Tutorials/Correlation_Tutorial_Element_Timeout.md

```diff
@@ -21,24 +21,31 @@ This tutorial consists of the following steps:
 
 - [Step 1: Deploy the 'Correlation KATA Switch Timeout' item from the DataMiner Catalog](#step-1-deploy-the-correlation-kata-switch-timeout-item-from-the-dataminer-catalog)
 - [Step 2: Create the Correlation rule](#step-2-create-the-correlation-rule)
-- [Step 3: Generate a timeout alarm](#step-3-generate-a-timeout-alarm)
-- [Step 4: Verify whether the rule actions have been executed](#step-4-verify-whether-the-rule-actions-have-been-executed)
+- [Step 3: Enable the Correlation Rule](#step-3-enable-the-correlation-rule)
+- [Step 4: Generate a timeout alarm](#step-4-generate-a-timeout-alarm)
+- [Step 5: Verify whether the rule actions have been executed](#step-5-verify-whether-the-rule-actions-have-been-executed)
 
 ## Step 1: Deploy the 'Correlation KATA Switch Timeout' item from the DataMiner Catalog
 
 1. Look up the 'Correlation KATA Switch Timeout' item in the DataMiner Catalog.
 
 1. Deploy the catalog item to your DataMiner Agent by clicking the *Deploy* button.
 
+After deploying, an automation script will be provided and an element named "Correlation KATA Switch Timeout - Network Switch" will be created. This element can be found in the Surveyor at *DataMiner Catalog* > *Correlation KATA* > *Network Switch Timeout*.
```

Comment:
'will have been created' or something? (both for the automation script and for the alarm, now it sounds like the automation script will be handed to you after you have deployed the package or something)